Comp354Project/src/main/java/com/comp354project/Comp354Project/repository/AccountRepository.java

```diff
@@ -6,5 +6,6 @@
 import java.util.List;
 
 public interface AccountRepository extends CrudRepository<Account, Integer> {
-    public Account findByEmail(String email);
+    List<Account> findByEmail(String email);
```

Comment:
I think this should not be a list, and indeed be a single Account, since the email is unique. any idea why this is a list ?

Reply:
Yes it should not be a list for our implementation, sadly it has to be in order for spring to abstract the implementation. All queries based on fields must return a list of entities. It return null otherwise. We could implement the function ourselves and make it return a single entity, but didn't think it was worth the time.

Reply:
Disregard that. I was wrong. Changing my implementation.